src/libcharon/plugins/kernel_super/kernel_super_ipsec.c

```diff
@@ -756,7 +778,32 @@ static status_t add_policy_internal(private_kernel_super_ipsec_t* this,
 				}
 
 				*should_add_routes = true;
+
+				if (IKE_SUPER_PEER_INFO_TYPE_TUNNEL == super_peer_info->get_type(super_peer_info)) {
```

Comment:
Similar to the comment in `write_terminator_tunnel_ike_sa_uptime()`, this write should be set for configured tunnels and not terminator tunnels.
Note that `write_tunnel_oper_items()` should still be written only for terminator tunnels